climpred/comparisons.py

```diff
@@ -1,9 +1,21 @@
+import dask
 import numpy as np
 import xarray as xr
 
 from .checks import has_dims, has_min_len
 from .exceptions import DimensionError
 
+# import error
+# from .utils import _transpose_and_rechunk_to
+
+
+def _transpose_and_rechunk_to(new_chunk_ds, ori_chunk_ds):
+    """Chunk xr.object `new_chunk_ds` as another xr.object `ori_chunk_ds`.
```

Comment:
Can you clean this up with Args, etc.?

Reply:
I understand you want to have this standardized but the one-liner says it all.